vars/withGCPEnv.groovy

```diff
@@ -26,7 +26,7 @@ def call(Map args = [:], Closure body) {
   def credentialsId = args.containsKey('credentialsId') ? args.credentialsId : ''
   def secret = args.containsKey('secret') ? args.secret : ''
 
-  if (!credentialsId.trim() && !secret.trim()) {
```

Comment:
Fixes 
<img width="485" alt="image" src="https://user-images.githubusercontent.com/2871786/155127684-cce29fd6-af84-4ca2-ae4a-558008118798.png">


then errors is only shown in the console output:

<img width="1099" alt="image" src="https://user-images.githubusercontent.com/2871786/155127744-b447f7ee-c4d9-471c-b4a5-a4bd31b374dd.png">